components/playground/instance/tiflash_config.go

```diff
@@ -106,12 +106,19 @@ func writeTiFlashConfig(w io.Writer, version utils.Version, tcpPort, httpPort, s
 	logDir := fmt.Sprintf("%s/log", deployDir)
 	ip := AdvertiseHost(host)
 	var conf string
+
+	port := "#"
+	// For 7.1.0 or later, TiFlash HTTP service is removed, so we don't need to set http_port
+	if !tidbver.TiFlashNotNeedHTTPPortConfig(version.String()) {
```

Comment:
I heard that TiFlash donot need whole config file？

Reply:
just replace the config file with args, still need some configuration.